src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py

```diff
@@ -1121,6 +1121,7 @@ def get_node_settings(_flow_dag_path: Path):
             )
             assert get_node_settings(Path(source)) != get_node_settings(new_flow_dag_path)
 
+    @pytest.mark.skip("Raise Exception: Process terminated with exit code 4294967295 in windows CI")
```

Comment:
use @pytest.mark.skipif(sys.platform == "win32", reason="...") to filter platform?

Reply:
updated